apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/brev/behandling/Behandling.kt

```diff
@@ -107,7 +111,7 @@ fun Grunnlag.mapSpraak(): Spraak = with(this.sak) {
 
 fun List<Beregningsperiode>.hentUtbetaltBeloep(): Int {
     // TODO: Håndter grunnbeløpsendringer
-    return this.last().utbetaltBeloep
+    return this.last().utbetaltBeloep.value
```

Comment:
må ikke alle periodene være med her? mulig det ikke er relevant for akkurat endringen din men hvis ikke bør det sikkert være en oppgave på det

Reply:
Litt usikker på hva du tenker her? Denne uthentingen gir kun hva du får som utbetalt beløp på første utbetaling etter du får brevet. I brev vil det stå noe ala `Du får 1 234 kroner hver måned før skatt`